rmgpy/data/kinetics/family.py

```diff
@@ -73,7 +73,7 @@ class TemplateReaction(Reaction):
     `family`    ``str``                   The kinetics family that the reaction was created from.
     `estimator` ``str``                   Whether the kinetics came from rate rules or group additivity.
     `reverse`   :class:`TemplateReaction` The reverse reaction, for families that are their own reverse.
-    `isForward` ``bool``                  Whether the reaction was generated in the forward direction of the family.
+    `is_forward` ``bool``                  Whether the reaction was generated in the forward direction of the family.
```

Comment:
Since this is a character longer, I think you need to increase the size of the table as well. You'll need to add another `=` to the borders and also add a space to the other entries.

Reply:
done